contracts/Bond.sol

```diff
@@ -47,21 +49,22 @@ contract Bond is
     uint256 public convertibleRatio;
 
     /**
-        @notice this role permits the withdraw of collateral from the contract
-        @dev this is assigned to owner in `initialize`
-            the owner can assign other addresses with this role to enable their withdraw
+        @notice This role permits the withdraw of collateral from the contract
+        @dev This role is assigned to the owner upon bond creation who can also
+            assign this role to other addresses to enable their withdraw.
+            
     */
     bytes32 public constant WITHDRAW_ROLE = keccak256("WITHDRAW_ROLE");
 
-    /// @dev used to confirm the bond has not yet matured
+    /// @dev Used to confirm the bond has not yet matured
```

Comment:
How did you decide when to add the periods? seems like a lot of things are missing them still